cpp/src/Ice/SSL/SecureTransportTransceiverI.cpp

```diff
@@ -239,6 +239,14 @@ Ice::SSL::SecureTransport::TransceiverI::initialize(IceInternal::Buffer& readBuf
                     {
                         continue; // Call SSLHandshake to resume the handshake.
                     }
+                    else
+                    {
+                        throw SecurityException(
+                            __FILE__,
+                            __LINE__,
+                            "IceSSL: certificate verification failed. the certificate was explicitly rejected by the "
```

Comment:
I would remove the "explicitly". And capitalize after the period: The certificate was rejected...